encryption/src/test/java/org/apache/solr/encryption/EncryptionTestUtil.java

```diff
@@ -130,13 +143,29 @@ public EncryptionStatus encrypt(String keyId) throws Exception {
     params.set(PARAM_KEY_ID, keyId);
     params.set(PARAM_TENANT_ID, TENANT_ID);
     params.set(PARAM_ENCRYPTION_KEY_BLOB, generateKeyBlob(keyId));
+    if (shouldDistributeEncryptRequest()) {
+      return encryptDistrib(params);
+    }
     GenericSolrRequest encryptRequest = new GenericSolrRequest(SolrRequest.METHOD.GET, "/admin/encrypt", params);
     EncryptionStatus encryptionStatus = new EncryptionStatus();
     forAllReplicas(replica -> {
       NamedList<Object> response = requestCore(encryptRequest, replica);
+      EncryptionRequestHandler.State state = EncryptionRequestHandler.State.fromValue(response.get(ENCRYPTION_STATE).toString());
       encryptionStatus.success &= response.get(STATUS).equals(STATUS_SUCCESS);
-      encryptionStatus.complete &= response.get(ENCRYPTION_STATE).equals(STATE_COMPLETE);
-    });
+      encryptionStatus.complete &= state == EncryptionRequestHandler.State.COMPLETE;
+    }, false);
+    return encryptionStatus;
+  }
+
+  private EncryptionStatus encryptDistrib(ModifiableSolrParams params) throws SolrServerException, IOException {
+    params.set(DISTRIB, "true");
```

Comment:
icky to manipulate the input arg but I can let it slide.  I wish there was a Solr one-liner to wrap SolrParams with a default.  SolrParams.wrapDefaults is a technique.

Reply:
Thanks, I didn't know the SolrParams.wrapDefaults technique.